indexer/indexer_tests/interval_index_test.cpp

```diff
@@ -12,11 +12,13 @@ namespace
 {
 struct CellIdFeaturePairForTest
 {
-  CellIdFeaturePairForTest(uint64_t cell, uint32_t feature) : m_Cell(cell), m_Feature(feature) {}
-  uint64_t GetCell() const { return m_Cell; }
-  uint32_t GetFeature() const { return m_Feature; }
-  uint64_t m_Cell;
-  uint32_t m_Feature;
+  using ValueType = uint32_t;
+
+  CellIdFeaturePairForTest(uint64_t cell, uint32_t value) : m_cell(cell), m_value(value) {}
+  uint64_t GetCell() const { return m_cell; }
+  uint32_t GetValue() const { return m_value; }
+  uint64_t m_cell;
```

Comment:
Please, insert a blank line between getters and fields.

Reply:
done